docs/apps/MinimalApps/Files/Context_menu.mdx

```diff
@@ -0,0 +1,10 @@
+---
+sidebar_position: 11
+---
+# Context menu based on the type of file
+
+![files 2.png](https://spectacled-wheel-852.notion.site/image/https%3A%2F%2Fs3-us-west-2.amazonaws.com%2Fsecure.notion-static.com%2Fb135d559-fa84-4036-8fa8-c2e3fd3372f3%2Ffiles_2.png?id=61a0805a-b1db-426c-a28a-daf28c51db61&table=block&spaceId=7a45c1be-b13e-401f-8da8-83b3f1ea585f&width=2000&userId=&cache=v2)
```

Comment:
Please don't use Notion as a CDN — if you need pictures, add them to the `static` folder.